plugins/decklink/decklink-device-instance.cpp

```diff
@@ -83,15 +95,18 @@ void DeckLinkDeviceInstance::HandleAudioPacket(
 	currentPacket.frames      = frameCount;
 	currentPacket.timestamp   = timestamp;
 
-	if (!ISSTEREO(channelFormat)) {
+	int maxdevicechannel = device->GetMaxChannel();
+
+	if ((channelFormat != SPEAKERS_UNKNOWN)&&(channelFormat != SPEAKERS_MONO)&&
```

Comment:
Style:  Add a single space on each side of the `&&` operators.  See also, next line.